src/components/ReportActionItem/IOUAction.js

```diff
@@ -24,6 +27,18 @@ const propTypes = {
         /** The participants of this report */
         participants: PropTypes.arrayOf(PropTypes.string),
     }),
+
+    /** Session of currently logged in user */
+    session: PropTypes.shape({
+        email: PropTypes.string.isRequired,
+    }).isRequired,
```

Comment:
Both the `session` and `personalDetails` props should not be required, and should default to `{}` (because providing props through Onyx is asynchronous). Of course, you'll need to make sure your code that uses those props is null-safe.

Reply:
Updated